documentation/status-logging.md

```diff
@@ -0,0 +1,55 @@
+# Status logging
+
+TK-status can periodically log status data to a file in JSON format.
+
+To take advantage of this you must do two things:
+
+1. Enable logging through you TK app's configuration. For example, to log every 2 seconds, your configuration might look like this:
```

Comment:
Although I suppose it might be good to point out that you can use fractions of a minute if you really want more granularity, like .5 to achieve 30 seconds.